backend/routes/mentorship.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const router = express.Router();
+const { Mentorship } = require("../database");
+
+// Routes
+
+router.use((request, response, next) => {
+  response.header("Access-Control-Allow-Origin", "*");
+  response.header(
+    "Access-Control-Allow-Headers",
+    "Origin, X-Requested-With, Content-Type, Accept"
+  );
+  next();
+});
+
+router.get("/", (request, response, next) => {
+  Mentorship.getAll()
+    .then(result => {
+      response.status(200).json(result);
+    })
+    .catch(error => {
+      response.status(400).json(error);
```

Comment:
Do you think "Unable to fetch Mentorship Pairs" would be a good one?

Reply:
Yeah that makes sense